test/exe/Main.hs

```diff
@@ -28,7 +29,56 @@ import Development.IDE.Test
 import Development.IDE.Test.Runfiles
 import Development.IDE.Types.Location
 import Language.LSP.Test
-import Language.LSP.Types hiding (mkRange)
+import Language.LSP.Types (
```

Comment:
maybe worth either qualifying the import or using `hiding`?  Definitely doesn’t have to be in this PR.

Reply:
for sure! I went with the explicit import list to avoid touching the code too much, but ideally I'd prefer to just have everything from this module qualified